docs/components/InputText/Web.stories.tsx

```diff
@@ -16,7 +16,9 @@ export default {
 } as ComponentMeta<typeof InputText>;
 
 const BasicTemplate: ComponentStory<typeof InputText> = args => {
-  return <InputText {...args} />;
+  const [value, setValue] = React.useState("");
+
+  return <InputText {...args} value={value} onChange={setValue} version={2} />;
```

Comment:
Are the changes in this file meant to be cleaned up?
The Readonly `InputText` story is broken again

Reply:
Thanks for catching this, I think something went wrong resolving conflicts